packages/components/src/templates/next/layouts/Skeleton/Skeleton.tsx

```diff
@@ -111,7 +112,9 @@ export const Skeleton = ({
       )}
 
       {/* Ensures that the webchat widget only loads after the page has loaded */}
+      {/* Note: did not account for both being added to the config as it's a very unlikely scenario and there's "correct" way to handle this */}
       {site.vica && <VicaWidget site={site} {...site.vica} />}
+      {site.wizgov && <WizgovWidget site={site} {...site.wizgov} />}
```

Comment:
just double checking - woudl having both `vica` and `wizgov` cause the widgets to overlay each other?

Reply:
probably! but given that this is admin-only and most site owners will only want 1, the risk is very minimal - also dom't think there's a "right" way to handle it. Disabling both or causing build to fail sounds off too

Reply:
i was thinking we choose `wizgov` if it exists, otherwise vica but i think it's rare enough that we can leave it as is